core/src/container.cpp

```diff
@@ -201,6 +201,9 @@ bool ContainerBase::insert(Stage::pointer&& stage, int before) {
 		return false;
 	}
 
+	if (!this->controllers().empty())
```

Comment:
This could just pass those generic trajectory properties (instead of having to add additional calls for each new feature)